cpp/src/aztec/honk/pcs/gemini/gemini.test.cpp

```diff
@@ -15,24 +15,26 @@ TYPED_TEST(GeminiTest, single)
     using MLEOpeningClaim = MLEOpeningClaim<TypeParam>;
 
     const size_t n = 16;
-    const size_t m = 4; // = log(n)
+    const size_t log_n = 4;
 
-    auto u = this->random_evaluation_point(m);
+    auto u = this->random_evaluation_point(log_n);
     auto poly = this->random_polynomial(n);
     auto commitment = this->commit(poly);
     auto eval = poly.evaluate_mle(u);
 
     // create opening claim
     auto claims = { MLEOpeningClaim{ commitment, eval } };
 
-    this->consume(u);
+    using Transcript = transcript::StandardTranscript;
```

Comment:
Is it hard to avoid duplicating this?

Reply:
No, and the same goes for all of the `using`s in these tests. I thought about moving them out but thought maybe Adrian did it this way for clarity or something. Either way is fine with me